### PR TITLE
fix: Saving stripe client test id

### LINKED ISSUE #3454: Stripe Client ID is not being saved
https://open-event-fe.netlify.com/admin/settings/payment-gateway

Stripe Test Client ID is not getting saved, hence cannot enable stripe

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #3454

### CODE REVIEW

**Comment by kushthedude** [comment]
@shreyanshdwivedi @uds5501 Please review

File: app/models/setting.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ export default ModelBase.extend({
   stripeClientId             : attr('string'),
   stripeSecretKey            : attr('string'),
   stripePublishableKey       : attr('string'),
-  stripeTestCliendId         : attr('string'),
```
Comment (iamareebjamal) [inline app/models/setting.js]: Why this passed in review?

Comment (mrsaicharan1) [reply app/models/setting.js]: @iamareebjamal Typo mistake.

Comment (kushthedude) [reply app/models/setting.js]: > Why this passed in review?

Might the PR be of large size, Thats why we would have missed noticing the error.

Comment (CosmicCoder96) [reply app/models/setting.js]: Why does anything pass in review indeed 😢

**Pull request merged**